extensions/amp-list/0.1/test/integration/test-amp-list.js

```diff
@@ -20,8 +20,6 @@ import {poll} from '../../../../../testing/iframe';
 const TIMEOUT = 15000;
 
 describe('amp-list (integration)', function() {
```

Comment:
Did you mean to remove this?

Reply:
yes. as we right now by default set timeout to 10s on Travis & 5s locally.